src/ui/ResponsiveComponents/ResponsiveTabs.ts

```diff
@@ -373,12 +373,11 @@ export class ResponsiveTabs implements IResponsiveComponent {
   private fromDropdownToTabSection(tab: Dom) {
     const lastTabInTabSection = last(this.tabsInTabSection);
     if (!lastTabInTabSection) {
-      this.tabSection.prepend(tab.el);
+      return this.tabSection.prepend(tab.el);
```

Comment:
Can you add a comment or such on why this return is important.

Reply:
I think it's pretty straightforward. If `tabsInTabSection` is not defined/ doesn't exist, why would you run through the code that checks it's index and inserts the current `tab` before/after it? Even more so that we insert the current `tab` in the section on line 376.

Reply:
Instead of returning the `this.tabSection.prepend(tab.el)` can we have the return statement on a new line ?

Reply:
Sure